src/main/java/com/serenitydojo/Cat.java

```diff
@@ -0,0 +1,37 @@
+package com.serenitydojo;
+
+public class Cat extends Pet {
+//    private String name;
+    private String favroriteFood;
+    private int age;
+
+    public Cat(String name,String favroriteFood,int age) {
+        super(name);
+//        this.name = name;
+        this.favroriteFood = favroriteFood;
+        this.age = age;
+    }
+//    public String getName() {
```

Comment:
it's usually a good idea to just remove comments. they just add clutter to the code.